oak/server/oak_runtime.cc

```diff
@@ -68,81 +112,36 @@ asylo::Status OakRuntime::Start() {
   LOG(INFO) << "Starting runtime";
 
   // Start the gRPC pseudo-node.
-  grpc_node_->Start();
+  if (grpc_node_ != nullptr) {
```

Comment:
This also seems very manual, we should just put all the nodes (and pseudo-nodes) in a map, iterate and start all of them. WDYT?

Reply:
Yep, planning to do something along these lines later on.